CONTRIBUTING.md

```diff
@@ -276,11 +277,12 @@ Both issue lists are sorted by total number of comments. While not perfect, numb
 
 #### Local development
 
-1. [Node.js](https://nodejs.org/en/) and [Git](https://git-scm.com/) and [Yarn](https://yarnpkg.com/)
+1. [Node.js](https://nodejs.org/en/) and [Git](https://git-scm.com/) and [Yarn](https://yarnpkg.com/) (version 3 - head over to https://yarnpkg.com/getting-started/install for instructions on how to install the latest yarn version).
 need to be installed on your machine.
 2. Install dependencies by executing `yarn` inside the project folder.
-3. Start up the local development server with `yarn start`.
-4. Now head over to [localhost:8080](http://localhost:8080) - you shoud see the Liquid docs site (on first build it may take a while until it shows up - relax and be patient).  
+3. Run `yarn build`. This step is necessary before first start up of the local dev server (it generates required type definitions within the src/liquid folder).
+4. Start up the local development server with `yarn start`.
+5. Now head over to [localhost:8080](http://localhost:8080) - you shoud see the Liquid docs site (on first build it may take a while until it shows up - relax and be patient).  
```

Comment:
```suggestion
5. Now head over to [localhost:8080](http://localhost:8080) - you shoud see the Liquid docs site.  
```
I'm not sure if the last part is still true. But I think we should leave it out anyway.